### ISSUE TITLE
Services.pm::get_darwin_equiv() assumes macOS v10.x

### ISSUE THREAD

**Issue by nieder** [fink/fink#210]
https://github.com/TheSin-/fink/blob/1e5752c85ad8bedb91b3fd48a6bddef375d036a6/perlmod/Fink/Services.pm#L1435-L1440

```perl
sub get_darwin_equiv {
	my %darwin_osx = (
		'1' => '10.0',
	);
	my $kernel_vers = get_kernel_vers();
	return $darwin_osx{$kernel_vers} || '10.' . ($kernel_vers-4);
}
```
This assumes that the kernel version via uname is 4 integers ahead of the the macOS minor version (darwin19 -> macOS 10.15). This breaks with macOS 11.0, which is darwin20.

Conditionalize to `if kernel_vers <=19, then [above code]; else $darwin_osx = $kernel_vers-9` ?

**Comment by nieder** [comment]
oops. I need to subtract for the minor version. So `$kernel_vers-20`, then.

**Comment by nieder** [comment]
Would have been fixed in 5735fa9

**Comment by nieder** [comment]
Fixed
